utils/check_publications_status.py

```diff
@@ -6,6 +6,7 @@
 """
```

Comment:
I'm not able to run this script successfully; are you able to run it?

Reply:
The error I get is:

```
ModuleNotFoundError: No module named 'attribute_dictionary'
```

which is most likely due to `attribute_dictionary.py` being in a different folder.  Unfortunately, Python has a hard time importing modules when they're located in an upstream and/or different folder from the main script.

One workaround we could do is to convert the dictionaries in `attribute_dictionary.py` into their own JSON file; that way, we could read them in as dictionaries (with the `json` module) regardless of where they are in the repo.  What do you think?

Reply:
Outside of that error, I also get a new error in the status_check function when trying to concat the rows together:

```
...
TypeError: cannot concatenate object of type '<class 'list'>'; only Series and DataFrame objs are valid
```

Did you encounter something like this?

Reply:
Hmm... Yeah I tested it yesterday and it worked for me. I will try again today and try to fix it!

Reply:
It works for me. But it sounds like we need to change it because it doesn't work for you. I am not sure how to do your JSON suggestion, but I'm happy to try it out and see if it works. How do we do it?

Reply:
I am not sure about the concat error. I don't get this either and you appear to concat the data frames before I added the edits in the script, so I'm not sure what would be causing it.

Reply:
Oh! I think my issue was that I was running the script from the parent folder, e.g. `mc2-center-dcc` whereas you were running from within the `mc2-center-dcc/utils/`.

I no longer get any of the errors noted above..... 😅

Reply:
Oh good!